internal/controller/dnsrecord_controller.go

```diff
@@ -194,26 +207,39 @@ func (r *DNSRecordReconciler) publishRecord(ctx context.Context, dnsRecord *v1al
 	}
 	err := r.Get(ctx, client.ObjectKeyFromObject(managedZone), managedZone, &client.GetOptions{})
 	if err != nil {
-		return err
+		return "0s", err
```

Comment:
so kinda wondering again why we went for a string here?

Reply:
updated